shared/types.ts

```diff
@@ -0,0 +1,14 @@
+export type Graphql = {
```

Comment:
Sharing the input/output type for different graphql queries, haven't tried to link input to output based on type, maybe can look at this later